staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/controller.go

```diff
@@ -157,22 +128,21 @@ func (c *AggregationController) sync(key string) (syncAction, error) {
 
 // AddAPIService adds a new API Service to OpenAPI Aggregation.
 func (c *AggregationController) AddAPIService(handler http.Handler, apiService *v1.APIService) {
-	if apiService.Spec.Service == nil {
```

Comment:
This line is needed to differentiate local and non-local apiservices. Otherwise built-ins and CRDs get added twice.